src/visualizers/VisualizerModFill.ts

```diff
@@ -46,14 +46,16 @@ class VizModFill extends VisualizerDefault implements VisualizerInterface {
 	drawNew(num: number, seq: SequenceInterface) {
 		const black = this.sketch.color(0);
 		this.sketch.fill(black);
-		let i: number;
-		let j;
-		for (let mod = 1; mod <= this.settings.modDimension; mod++) {
+		for (let mod = 1n;
+			mod <= BigInt(this.settings.modDimension);
+			mod++)
+		{
 			const el = seq.getElement(num);
-			if(el === undefined) break;
-			i = el % mod;
-			j = mod - 1;
-			this.sketch.rect(j * this.rectWidth, this.sketch.height - (i + 1) * this.rectHeight, this.rectWidth, this.rectHeight);
+			const xpos = Number(mod-1n) * this.rectWidth;
```

Comment:
I would have just been verbose and named these `xPosition` and `yPosition`. Modern text editors and IDE's should have some autocomplete feature, so I think being verbose when naming variables is acceptable.

EDIT 1: Same goes for `el`, but I take it @gwhitney didn't name `el`.

EDIT 2: Now that I think about the code base, there are tons of variables like this. Maybe we should discuss naming at some point.

Reply:
My basic view is that utility variables with textually small scope (easily fitting on screen) can have abbreviated names as long as the code is clear, and that insisting on long variable names, given the line character limit, can sometimes waste relatively precious vertical space. I agree that variables that are used in disparate places in the code must have descriptive names that are clear stand-alone.

In this particular case, the _only_ reason the variables are being introduced is to abbreviate long expressions that could be placed in-line, but that would make for a much harder-to-read gargantuan line. So since the purpose is to abbreviate, it makes sense to me to use abbreviated names. I will just switch to `x` and `y` -- I often avoid those because they are just so generic, I'm slightly worried about unintended name clashes, but I think in this particular case they are OK in all respects.

Reply:
I changed el to s while I was at it, which I presume is less objectionable.